### PR TITLE
new build for kebab 20.09.2024.

### PR DESCRIPTION
**Pull request by bojantanasic**
first official build for kebab

### CODE REVIEW

File: kebab.json [hunk 1]
```diff
@@ -4,22 +4,22 @@
 			"maintainer": "",
```
Comment (gwolf2u) [inline kebab.json]: Missing info

File: kebab.json [hunk 2]
```diff
@@ -4,22 +4,22 @@
...
 			"firmware": "",
 			"modem": "",
 			"bootloader": "",
-			"recovery": "https://sourceforge.net/projects/crdroid/files/kebab/10.x/recovery/",
-			"paypal": "https://paypal.me/saurabhparab007",
-			"telegram": "https://t.me/crdroidop8",
+			"recovery": "https://sourceforge.net/projects/crdroid10/files/",
+			"paypal": "https://www.paypal.com/paypalme/bojantanasic",
+			"telegram": "https://t.me/crDroid10",
 			"dt": "https://github.com/crdroidandroid/android_device_oneplus_kebab",
```
Comment (gwolf2u) [inline kebab.json:23]: please push dt and kernel sources

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by bojantanasic** [comment]
now everything is ok, I`m change what you requested

File: kebab.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
-			"recovery": "https://sourceforge.net/projects/crdroid/files/kebab/10.x/recovery/",
-			"paypal": "https://paypal.me/saurabhparab007",
-			"telegram": "https://t.me/crdroidop8",
-			"dt": "https://github.com/crdroidandroid/android_device_oneplus_kebab",
-			"common-dt": "https://github.com/crdroidandroid/android_device_oneplus_sm8250-common",
-			"kernel": "https://github.com/crdroidandroid/android_kernel_oneplus_sm8250"
+			"recovery": "https://sourceforge.net/projects/crdroid10/files/",
+			"paypal": "https://www.paypal.com/paypalme/bojantanasic",
+			"telegram": "https://t.me/crDroid10",
+			"dt": "https://github.com/bojantanasic/device_oneplus_kebab",
```
Comment (gwolf2u) [inline kebab.json]: dt needs to be hosted on crdroid git
entries were ok, but source was not pushed

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: kebab.json (hunk 3 above)
Comment (bojantanasic) [reply kebab.json]: now it`s pushed on a branch 14

Comment (AnyOne06) [reply kebab.json]: This Branches looks horrible 
Please clean it
Also why not push it to 14-qpr3.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**